### PR TITLE
feat: added tile command to cli

### PR DESCRIPTION
**Pull request by stanislavulrych**
Added `tile` command, which can be used for tiling multiple stickers for printing in sticker papers.

### CODE REVIEW

File: pdfly/tiles.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+            Omitting one position by using `-` (also omitted -x 2 -y 2 as those are by default).
+    
+            A |        B |       C |                   | A
+            --|--      --|--     --|--     ===>      --|--
+              |          |         |                 B | C
+              
+"""
+
+# Stanislav Ulrych <stanislav.ulrych@gmail.com>.
```
Comment (MartinThoma) [inline pdfly/tiles.py:39]: Please don't add such comments

File: pdfly/tiles.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        output_fh = open(output, "wb")
+    else:
+        sys.stdout.flush()
+        output_fh = os.fdopen(sys.stdout.fileno(), "wb")
+
+    xsize = PaperSize.A4.width
+    ysize = PaperSize.A4.height
+
+    writer = PdfWriter()
+    destPage = writer.add_blank_page(width=xsize, height=ysize)
```
Comment (MartinThoma) [inline pdfly/tiles.py:63]: Please follow [PEP-8](https://peps.python.org/pep-0008/):

```suggestion
    dest_page = writer.add_blank_page(width=xsize, height=ysize)
```

File: pdfly/tiles.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                destPage.merge_page(p)
+
+        writer.write(output_fh)
+    except Exception:
+        print(traceback.format_exc(), file=sys.stderr)
+        print(f"Error while reading {f}", file=sys.stderr)
+        sys.exit(1)
+    finally:
+        output_fh.close()
+    # In 3.0, input files must stay open until output is written.
```
Comment (MartinThoma) [inline pdfly/tiles.py:87]: Isn't that true for Python 2.x as well ... and any programming language for that matter?

```suggestion
```

File: pdfly/tiles.py [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+            if f == "-":
+                continue
+
+            reader = PdfReader(f)
+            for p in reader.pages:
+                t = {
+                    "tx": (i % xcount)*(xsize/xcount),
+                    "ty": -(i // xcount)*(ysize/ycount) 
+                }
+                print(t)
```
Comment (MartinThoma) [inline pdfly/tiles.py:76]:
```suggestion
```

**Comment by stanislavulrych** [comment]
Comments addressed.

`pdfly tiles -o ./out.pdf ./in.pdf - ./in.pdf ./in.pdf`

[out.pdf](https://github.com/py-pdf/pdfly/files/13273683/out.pdf)
[in.pdf](https://github.com/py-pdf/pdfly/files/13273684/in.pdf)

**Pull request closed without merging**